doc/examples/transform/plot_masked_register_translation.py

```diff
@@ -0,0 +1,68 @@
+"""
+===================================
+Masked Normalized Cross-Correlation
+===================================
+
+In this example, we use the masked normalized cross-correlation to identify the relative shift
+between two similar images containing invalid data.
+
+In this case, the images cannot simply be masked before computing the cross-correlation, 
+as the masks will influence the computation. The influence of the masks must be removed from
+the cross-correlation, as is described in [1]_.
+
+In this example, we register the translation between two images. However, one of the 
+images has about 25% of the pixels which are corrupted.
+
+.. [1] D. Padfield, "Masked object registration in the Fourier domain" 
+       IEEE Transactions on Image Processing (2012). :DOI:`10.1109/TIP.2011.2181402`
+
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+from skimage import data
+from skimage.feature import masked_register_translation
+from scipy.ndimage import fourier_shift
+
+image = data.camera()
+shift = (-22, 13)
+
+# Define areas of the image which are invalid
```

Comment:
If you want this block to appear as a text block in the rendered html, you need to add a line of "######" above (see for example ``plot_local_binary_pattern.py``).

Reply:
Good trick, thanks